src/devices/Ak8963/README.md

```diff
@@ -0,0 +1,115 @@
+# AK8963 - Magnetometer
+
+The AK8963 is a magnetometer that can be controlled either thru I2C either thru SPI. It is present in other sensors like the [MPU9250](../Mpu9250/README.md). This implementation fully supports the I2C mode and the usage thru the MPU9250. It does not support SPI.
+
+## Usage
+
+You can find an example in the [sample](./samples/ak8963.sample.cs) directory. Usage is straight forward including the possibility to have a calibration.
```

Comment:
fix casing of the path (for linux users)

Reply:
path is correct, file is really lowercase when I created it. Just checked.

Reply:
my bad, I could swear I thought `samples` was capitalized 😄